tests/unit/browser/test_qutehistory.py

```diff
@@ -0,0 +1,117 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2017 Imran Sobir
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+from PyQt5.QtCore import QUrl
+from qutebrowser.browser import history, qutescheme
+from qutebrowser.utils import objreg
+import datetime
+import pytest
+import tempfile
+
+
+class TestHistoryHandler:
```

Comment:
Please add an empty line before class docstrings (IIRC pep8/pep257 says something different nowadays, but it used to be that way, and all other qutebrowser code is still using that convention).